src/acceptance-test/java/de/zalando/aruha/nakadi/webservice/hila/StreamBatch.java

```diff
@@ -0,0 +1,48 @@
+package de.zalando.aruha.nakadi.webservice.hila;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.collect.ImmutableList;
+import de.zalando.aruha.nakadi.domain.Cursor;
+
+import javax.annotation.concurrent.Immutable;
+import java.util.List;
+import java.util.Map;
+
+import static java.util.Collections.unmodifiableList;
+
+@Immutable
+public class StreamBatch {
+
+    private Cursor cursor;
+    private List<Map> events = ImmutableList.of();
+
+    public StreamBatch(@JsonProperty("cursor") final Cursor cursor, @JsonProperty("events") final List<Map> events) {
```

Comment:
For EventType deserialisation happens using setters, which doesn't require any additional markup.
For this class there are no setters as I wanted it to be immutable, so the deserialisation happens using constructor and Jackson needs to know which json property should go to which constructor parameter.